keps/sig-storage/695-skip-permission-change/README.md

```diff
@@ -34,7 +34,7 @@
 - [x] (R) Design details are appropriately documented
 - [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [x] (R) Graduation criteria is in place
-- [ ] (R) Production readiness review completed
+- [x] (R) Production readiness review completed
```

Comment:
@gnufied can you also update the PRR answer with the new fsgroup mount latency metric?

Reply:
done.